pre_commit_hooks/check_executables_have_shebangs.py

```diff
@@ -8,23 +8,36 @@
 from typing import NamedTuple
 from typing import Sequence
 
+from pre_commit_hooks.util import CalledProcessError
 from pre_commit_hooks.util import cmd_output
 from pre_commit_hooks.util import zsplit
 
 EXECUTABLE_VALUES = frozenset(('1', '3', '5', '7'))
 
 
 def check_executables(paths: list[str]) -> int:
-    if sys.platform == 'win32':  # pragma: win32 cover
```

Comment:
feels to me like this should be a 2-line patch here instead of rewriting half the file

Reply:
the coverage issues you experienced _were caused by your mistake_

Reply:
I don't see a connection and just pushed a test commit on both PRs demonstrating that all of the pragma statements remain necessary in the current state of both branches. Again, if I'm missing something, please let me know, and I will gladly correct the mistake. I will, of course, take the test commits back down after you have a chance to respond.

Reply:
I don't really get what you don't understand -- if you make the two line patch and leave the current pragmas in place you'll pass coverage

Reply:
and if it doesn't pass coverage, that shows that your change is incorrect

Reply:
The patch I started from (`!= 'true'`) was not the same one you had in mind (the simpler `== 'false'`). Again though, the pragmas were correct at every iteration, and my initial bug was not merely a result of disabling a check that should have been left in place. The previous iteration was functionally correct if verbose.

Reply:
To clarify, I simplified it to use `== 'false'` instead.